src/main/java/com/moodybluez/enterprise/dto/Entry.java

```diff
@@ -4,7 +4,10 @@
 
 public @Data
 class Entry {
+    private int entryID;
```

Comment:
I added this ID attribute, because I think it would be helpful to get a entry by specific ID down the road, as "Entry" class is also major DTO in your project.